File: app/src/main/java/org/fossasia/openevent/general/model/EventList.java
package org.fossasia.openevent.general.model;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.databind.PropertyNamingStrategy;
import com.fasterxml.jackson.databind.annotation.JsonNaming;

import java.util.List;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * Created by harsimar on 20/05/18.
 */
@Data
@NoArgsConstructor
@AllArgsConstructor
@JsonNaming(PropertyNamingStrategy.KebabCaseStrategy.class)
public class EventList {
    @JsonProperty("data")
    private List<Event> eventList;

}

File: app/src/main/java/org/fossasia/openevent/general/model/EventList.kt
package org.fossasia.openevent.general.model

import com.fasterxml.jackson.annotation.JsonProperty
import com.fasterxml.jackson.databind.PropertyNamingStrategy
import com.fasterxml.jackson.databind.annotation.JsonNaming

@JsonNaming(PropertyNamingStrategy.KebabCaseStrategy::class)
data class EventList(
    @field:JsonProperty("data")
    val eventList: List<Event>
)
